### ISSUE TITLE
[BUG] while useing translate component 

### ISSUE THREAD

**Issue by gagore** [ShaneBeee/SkBee#217]
**Describe the bug**
A. It translates only into English
B. Error executing syntax

**To Reproduce**

```
A.
set {_a} to translate component from "item.minecraft.diamond_sword"
Please do it in a different language
It used to work normally But not now
```
```
B.
set {_b} to translate component from a diamond sword
Running the syntax results in this problem
```

**Expected behavior**
```
A. The language set by the client must come out
```
```
B. It has to be executed without errors
```

**Screenshots/Errors**

A.

![image](https://user-images.githubusercontent.com/93857579/180206104-b6902f27-fcc6-4d69-b402-70dcdb52724c.png)

```
B.
[20:44:11 ERROR]: #!#!
[20:44:11 ERROR]: #!#! [Skript] Severe Error:
[20:44:11 ERROR]: #!#!
[20:44:11 ERROR]: #!#! Something went horribly wrong with Skript.
[20:44:11 ERROR]: #!#! This issue is NOT your fault! You probably can't fix it yourself, either.
[20:44:11 ERROR]: #!#! It looks like you are using some plugin(s) that alter how Skript works (addons).
[20:44:11 ERROR]: #!#! Here is full list of them:
[20:44:11 ERROR]: #!#! skript-reflect v2.3 (https://github.com/TPGamesNL/skript-reflect) TranSKator v0.1.8 (https://discord.gg/G6ca9S5) MorkazSk v1.3 SkEtcR v3.2 skript-holo v1.0.2 (https://github.com/Blueyescat/skript-holo) SkBee v1.17.2 (https://github.com/ShaneBeee/SkBee) skript-yaml v1.4 SkQuery v4.1.7 Skellett v2.0.2 (https://forums.skunity.com/resources/skellett.24/) skript-db v1.3.4
[20:44:11 ERROR]: #!#! We could not identify which of those are specially related, so this might also be Skript issue.
[20:44:11 ERROR]: #!#! You should try disabling those plugins one by one, trying to find which one causes it.
[20:44:11 ERROR]: #!#! If the error doesn't disappear even after disabling all listed plugins, it is probably Skript issue.
[20:44:11 ERROR]: #!#! In that case, you will be given instruction on how should you report it.
[20:44:11 ERROR]: #!#! On the other hand, if the error disappears when disabling some plugin, report it to author of that plugin.
[20:44:11 ERROR]: #!#! Only if the author tells you to do so, report it to Skript's issue tracker.
[20:44:11 ERROR]: #!#!
[20:44:11 ERROR]: #!#! Stack trace:
[20:44:11 ERROR]: #!#! ch.njol.skript.SkriptAPIException: UnparsedLiterals must be converted before use
[20:44:11 ERROR]: #!#!     at Skript 2.6.3.jar//ch.njol.skript.lang.UnparsedLiteral.invalidAccessException(UnparsedLiteral.java:272)
[20:44:11 ERROR]: #!#!     at Skript 2.6.3.jar//ch.njol.skript.lang.UnparsedLiteral.getArray(UnparsedLiteral.java:292)
[20:44:11 ERROR]: #!#!     at SkBee-1.17.2.jar//com.shanebeestudios.skbee.elements.text.expressions.ExprTextComponent.get(ExprTextComponent.java:86)
[20:44:11 ERROR]: #!#!     at SkBee-1.17.2.jar//com.shanebeestudios.skbee.elements.text.expressions.ExprTextComponent.get(ExprTextComponent.java:38)
[20:44:11 ERROR]: #!#!     at Skript 2.6.3.jar//ch.njol.skript.lang.util.SimpleExpression.getArray(SimpleExpression.java:99)
[20:44:11 ERROR]: #!#!     at Skript 2.6.3.jar//ch.njol.skript.effects.EffChange.execute(EffChange.java:274)
[20:44:11 ERROR]: #!#!     at Skript 2.6.3.jar//ch.njol.skript.lang.Effect.run(Effect.java:50)
[20:44:11 ERROR]: #!#!     at Skript 2.6.3.jar//ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:61)
[20:44:11 ERROR]: #!#!     at Skript 2.6.3.jar//ch.njol.skript.lang.TriggerItem.walk(TriggerItem.java:89)
[20:44:11 ERROR]: #!#!     at Skript 2.6.3.jar//ch.njol.skript.lang.Trigger.execute(Trigger.java:56)
[20:44:11 ERROR]: #!#!     at Skript 2.6.3.jar//ch.njol.skript.command.ScriptCommand.execute2(ScriptCommand.java:286)
[20:44:11 ERROR]: #!#!     at Skript 2.6.3.jar//ch.njol.skript.command.ScriptCommand.lambda$execute$0(ScriptCommand.java:251)
[20:44:11 ERROR]: #!#!     at Skript 2.6.3.jar//ch.njol.skript.command.ScriptCommand.execute(ScriptCommand.java:256)
[20:44:11 ERROR]: #!#!     at Skript 2.6.3.jar//ch.njol.skript.command.Commands.handleCommand(Commands.java:261)
[20:44:11 ERROR]: #!#!     at Skript 2.6.3.jar//ch.njol.skript.command.Commands$1.onPlayerCommand(Commands.java:178)
[20:44:11 ERROR]: #!#!     at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37)
[20:44:11 ERROR]: #!#!     at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80)
[20:44:11 ERROR]: #!#!     at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
[20:44:11 ERROR]: #!#!     at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:669)
[20:44:11 ERROR]: #!#!     at net.minecraft.server.network.PlayerConnection.handleCommand(PlayerConnection.java:2368)
[20:44:11 ERROR]: #!#!     at net.minecraft.server.network.PlayerConnection.lambda$handleChatCommand$18(PlayerConnection.java:2147)
[20:44:11 ERROR]: #!#!     at net.minecraft.server.TickTask.run(TickTask.java:18)
[20:44:11 ERROR]: #!#!     at net.minecraft.util.thread.IAsyncTaskHandler.d(IAsyncTaskHandler.java:153)
[20:44:11 ERROR]: #!#!     at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.d(IAsyncTaskHandlerReentrant.java:24)
[20:44:11 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1358)
[20:44:11 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:183)
[20:44:11 ERROR]: #!#!     at net.minecraft.util.thread.IAsyncTaskHandler.x(IAsyncTaskHandler.java:126)
[20:44:11 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.bh(MinecraftServer.java:1335)
[20:44:11 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1328)
[20:44:11 ERROR]: #!#!     at net.minecraft.util.thread.IAsyncTaskHandler.c(IAsyncTaskHandler.java:136)
[20:44:11 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1306)
[20:44:11 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.v(MinecraftServer.java:1191)
[20:44:11 ERROR]: #!#!     at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:302)
[20:44:11 ERROR]: #!#!     at java.base/java.lang.Thread.run(Thread.java:833)
[20:44:11 ERROR]: #!#!
[20:44:11 ERROR]: #!#! Version Information:
[20:44:11 ERROR]: #!#!   Skript: 2.6.3 (latest)
[20:44:11 ERROR]: #!#!     Flavor: skriptlang-github
[20:44:11 ERROR]: #!#!     Date: 00:55:22.847925631
[20:44:11 ERROR]: #!#!   Bukkit: 1.19-R0.1-SNAPSHOT
[20:44:11 ERROR]: #!#!   Minecraft: 1.19
[20:44:11 ERROR]: #!#!   Java: 17.0.3.1 (Java HotSpot(TM) 64-Bit Server VM 17.0.3.1+2-LTS-6)
[20:44:11 ERROR]: #!#!   OS: Windows 10 amd64 10.0
[20:44:11 ERROR]: #!#!
[20:44:11 ERROR]: #!#! Server platform: Paper
[20:44:11 ERROR]: #!#!
[20:44:11 ERROR]: #!#! Current node: null
[20:44:11 ERROR]: #!#! Current item: set {_b} (as java.lang.Object) to a new translate component from 'a diamond sword'
[20:44:11 ERROR]: #!#! Current trigger: command /language (simple event) (backpack.sk, line 17)
[20:44:11 ERROR]: #!#!
[20:44:11 ERROR]: #!#! Thread: Server thread
[20:44:11 ERROR]: #!#!
[20:44:11 ERROR]: #!#! Language: english
[20:44:11 ERROR]: #!#! Link parse mode: DISABLED
[20:44:11 ERROR]: #!#!
[20:44:11 ERROR]: #!#! End of Error.
[20:44:11 ERROR]: #!#!
```

**Version Info:**
 - Provide output of **/skbee info**   
 **OR**
 - **SkBee Version:** 1.17.2
 - **Skript Version:** 2.6.3
 - **Server Version/Platform** paper1.19

**Additional context**
Thank you for your hard work

**Comment by ShaneBeee** [comment]
A) works perfect fine for me, I tried with both in game command as well as a script (using the pirate language)
<img width="582" alt="Screen Shot 2022-07-21 at 8 00 49 AM" src="https://user-images.githubusercontent.com/20008482/180246896-5f89441e-9f84-4f13-89cc-20da184788e6.png">

<img width="554" alt="Screen Shot 2022-07-21 at 8 00 14 AM" src="https://user-images.githubusercontent.com/20008482/180246884-dbe1c63c-a279-483e-8d66-53fd6fb0326a.png">

B) That is a bug that I'll fix

For the future, since these are 2 very different issues (ie: A is not an issue, B is an issue and very different) please report separately

**Comment by ShaneBeee** [comment]
b) fixed in 1.17.3
